test-cases/python/web-security-issues.py

```
@@ -0,0 +1,152 @@
+def create_user(user_data):
+    user = {
+        'email': user_data['email'],
+        'password': user_data['password'],
+        'role': user_data['role']
+    }
+    save_user(user)
+    return True
+
+@app.route('/set_cookie')
+def set_cookie():
+    response = make_response("Cookie set")
```

Ruff (0.12.2)

Undefined name `make_response`

(F821)